.github/workflows/check-broken-links.yml

```diff
@@ -11,6 +11,16 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Close Existing Broken Links Issue(s)
+        uses: actions/stale@v8
+        with:
+          repo-token: ${{secrets.GITHUB_TOKEN}}
+          only-issue-labels: automated issue,Broken links
+          # close-issue-message requires permission `issues: write`
+          close-issue-message: 'This issue is closed due to a new "Check Broken Links" run. If any links are still broken, they will be reported in a new issue: https://github.com/osmosis-labs/osmosis/issues?q=is%3Aissue+is%3Aopen+label%3A%22automated+issue%22+label%3A%22Broken+links%22'
```

Comment:
Is this link suppose to take us somewhere?

https://github.com/osmosis-labs/osmosis/issues?q=is%3Aissue+is%3Aopen+label%3A%22automated+issue%22+label%3A%22Broken+links%2

Reply:
Ah I think you're missing the last `2` in the link. It's a link to our open issues filtered with both the label `automated issue` and `Broken links`. So if there's a new issue generated, people can see it there.